src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/action_page.php"
+      method="get"
+      data-qa="big"
+    >
+      <input
+        class="input search bigsearch"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+      <img
+        src="/images/Search.svg"
+        alt="searchloop"
+        class="searchloop"
+      >
+    </form>
+    <form
+      action="/action_page.php"
```

Comment:
Here you add both base class and modifier. To get as base styles as modifier one
```suggestion
      class "form form--small"
      action="/action_page.php"
```